beacon-chain/rpc/eth/beacon/handlers_test.go

```diff
@@ -2604,8 +2612,10 @@ func TestServer_GetBlockRoot(t *testing.T) {
 		}
 		for _, tt := range tests {
 			t.Run(tt.name, func(t *testing.T) {
+				blockID:=tt.blockID
 				request := httptest.NewRequest(http.MethodGet, url, nil)
-				request = mux.SetURLVars(request, tt.blockID)
+				// Review needed 
+				request.PathValue(blockID["block_id"])
```

Comment:
Shouldn't this be `mux.SetPathValue` instead?

Reply:
You are correct fixed this